fast-cart/src/components/Navbar.js

```diff
@@ -0,0 +1,41 @@
+import { useMemo } from "react";
+import { Link } from "react-router-dom";
+import { useShopping } from "../context/ShoppingListContext";
+
+const Navbar = () => {
+
+    const shoppingList = useShopping();
+
+    const totalItems = useMemo(()=>{
+        var number = 0;
```

Comment:
avoid using _var_ instead use let/const, it can cause you many trouble related to scope issues

https://medium.com/@codingsam/awesome-javascript-no-more-var-working-title-999428999994